automation/src/test/scala/org/broadinstitute/dsde/firecloud/fixture/UserFixtures.scala

```diff
@@ -41,8 +44,18 @@ trait UserFixtures extends CleanUp { self: WebBrowserSpec with TestSuite =>
                                                 (testCode: (T) => Any)
                                                 (implicit webDriver: WebDriver): Unit = {
     withSignIn(user, {
-      new SignInPage(Config.FireCloud.baseUrl).open
-      executeScript(s"window.forceSignedIn('${user.makeAuthToken().value}')")
+      val openedPage: SignInPage = new SignInPage(Config.FireCloud.baseUrl).open
+      retry[T](1.seconds, 60.seconds) ({
```

Comment:
Because this times out after 60 seconds and inside we wait 30 seconds for sign-in to succeed, this will retry approximately 1 time. That's hard logic to follow, somewhat unstable depending on timing, and hard to reason about.

I'd prefer using the other form of `retry`:

```
retry(Seq.fill(2)(1.second)) ({
    ...
```

which inserts two 1-second intervals between tries; this will always try up to 3 times, regardless of how long each try takes.